src/pages/upload/index.tsx

```diff
@@ -116,20 +113,31 @@ const UploadWrapper = ({
           lessons: values.lessons,
         })
 
+        setSubmitResponse(response)
```

Comment:
capture the response from the sanity POST so that we can display a Success/Fail message from it.